debian/cloud-init-base.postinst

```diff
@@ -55,8 +55,8 @@ handle_preseed_maas() {
    local cfg_file="/etc/cloud/cloud.cfg.d/90_dpkg_maas.cfg"
    local md_url="" creds_all="" c_key="" t_key="" t_sec="" c_sec="";
 
-   db_get "cloud-init-base/maas-metadata-url" && md_url="$RET" || :
-   db_get "cloud-init-base/maas-metadata-credentials" && creds_all="$RET" || :
+   process_debconf_setting maas-metadata-url && md_url="$RET" || :
```

Comment:
This would fail since the function is not defined at this point in the script.

I assume that also means that this hasn't been tested?

Reply:
I ran through live upgrade tests here and recorded echo'd messages as well as changed/migrated settings values. Given the changes I just pushed though, I'll retest and report the successful output of an upgrade run.